packages/brand-components/src/Testimonial/index.tsx

```diff
@@ -17,13 +17,15 @@ export type Testimonial = {
 type TestimonialProps = {
   testimonial: Testimonial;
   size: 'small' | 'medium' | 'large';
+  fullWidth?: boolean;
```

Comment:
What do you all think about doing this instead?

```suggestion
  size: 'small' | 'medium' | 'large' | 'full';
```

Fewer props -> simpler code -> happier devs

Reply:
🆗  good idea